src/tests/actions/getUserInfoAction.test.js

```diff
@@ -0,0 +1,41 @@
+import moxios from 'moxios'
+import thunk from 'redux-thunk'
+import configureMockStore from 'redux-mock-store'
+import { fetchUserInfo } from '../../actions/getUserInfoAction'
+import { GET_USER_INFO } from '../../types'
+import userInfoResponse from '../../fixtures/userInfo'
+
+const middlewares = [thunk]
+const mockStore = configureMockStore(middlewares)
+let store
+
+describe('fetchUserInfo action', () => {
+  beforeEach(() => {
+    moxios.install()
+    store = mockStore({})
+  })
+
+  afterEach(() => {
+    moxios.uninstall()
+  })
+
+  it('fetches user info from an external api', async () => {
+    const expectedActions = [
+      { type: GET_USER_INFO, payload: userInfoResponse }
+    ]
+    moxios.stubRequest('/api/v1/users/1', {
+      status: 200
+    })
+
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent()
+      request.resolve({
+        data: { user: userInfoResponse }
+      })
+    })
+
+    await store.dispatch(fetchUserInfo()).then(() => {
+      expect(store.getActions()).toEqual(expectedActions)
```

Comment:
Is this test always passing? or does if you change the expectations it fails for the right reasons?

Reply:
it fails if you change the expectation to anything other than was it should be 
```
Expected value to equal:
      [[{"payload": {"activity": undefined, "authentications": undefined, "bio": undefined, "commitCountTotal": undefined, "contributions": undefined, "gravatarUrl": undefined, "hangouts": undefined, "karmaTotal": undefined, "membershipLength": undefined, "profile": undefined, "projects": undefined, "user": {"bio": "", "can_see_dashboard": false, "city": "Florianopolis", "country_code": "BR", "country_name": "Brazil", "created_at": "2015-01-26T23:19:00.575Z", "deleted_at": null, "display_email": null, "display_hire_me": null, "display_profile": true, "email": "mattwr18@gmail.com", "event_participation_count": 0, "first_name": "Matt", "github_profile_url": null, "id": 1, "last_name": "Tester", "latitude": 27.5949, "longitude": 48.5482, "receive_mailings": true, "region": "Santa Catarina", "skill_list": [], "slug": "matt-tester", "status_count": 0, "timezone_offset": -18000, "title_list": ["Mentor"], "updated_at": "2016-08-22T09:23:23.947Z", "youtube_id": null, "youtube_user_name": null}, "videos": undefined}, "type": "GET_USER_INFO"}]]
    Received:
      [{"payload": {"activity": undefined, "authentications": undefined, "bio": undefined, "commitCountTotal": undefined, "contributions": undefined, "gravatarUrl": undefined, "hangouts": undefined, "karmaTotal": undefined, "membershipLength": undefined, "profile": undefined, "projects": undefined, "user": {"bio": "", "can_see_dashboard": false, "city": "Florianopolis", "country_code": "BR", "country_name": "Brazil", "created_at": "2015-01-26T23:19:00.575Z", "deleted_at": null, "display_email": null, "display_hire_me": null, "display_profile": true, "email": "mattwr18@gmail.com", "event_participation_count": 0, "first_name": "Matt", "github_profile_url": null, "id": 1, "last_name": "Tester", "latitude": 27.5949, "longitude": 48.5482, "receive_mailings": true, "region": "Santa Catarina", "skill_list": [], "slug": "matt-tester", "status_count": 0, "timezone_offset": -18000, "title_list": ["Mentor"], "updated_at": "2016-08-22T09:23:23.947Z", "youtube_id": null, "youtube_user_name": null}, "videos": undefined}, "type": "GET_USER_INFO"}]
```